lib/syntax_tree/node.rb

```diff
@@ -2103,10 +2108,15 @@ class BlockVar < Node
     # [Array[ Comment | EmbDoc ]] the comments attached to this node
     attr_reader :comments
 
-    def initialize(params:, locals:, location:)
+    # [boolean] whether or not the variables are within pipes
+    attr_reader :pipe
+    alias pipe? pipe
```

Comment:
I kind of modeled this after the `inline` attribute on the `Comment` node. As far as I'm aware, for formatting and translating reasons we need to keep track of this, but let me know if you'd prefer a different approach.